app/react/Markdown/components/PublicForm.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Immutable from 'immutable';
+import { MetadataFormFields, validator } from 'app/Metadata';
+import { LocalForm, actions, Control } from 'react-redux-form';
+import { Captcha } from 'app/ReactReduxForms';
+import { Translate } from 'app/I18N';
+import { publicSubmit } from 'app/Uploads/actions/uploadsActions';
+import { bindActionCreators } from 'redux';
+import { FormGroup } from 'app/Forms';
+
+export class PublicForm extends Component {
```

Comment:
I like it that the test is using WrappedComponent to access the connected exported component.  It that is the case, why the need to export the base component?